### PR TITLE
Crypto account check

### PR DESCRIPTION
**Pull request by vimalan-deriv**
In this test case we would be creating all the available cryptocurrency in deriv and check for the confirmation message and also details displayed in the manage account page

### CODE REVIEW

File: cypress/e2e/wip/Crypto/crypto_account_check.cy.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      cy.findByText('Success!')
+      cy.findByText('Make a deposit now to start trading.')
+      cryptoconfig.elements.maybelater().should('be.visible').click()
+    }})
+  return crypto
+};
+
+describe('QATEST-707 - Create crypto account', () => {
+  beforeEach(() => {
+    cy.c_login()
```
Comment (abuzar-deriv) [inline cypress/e2e/wip/Crypto/crypto_account_check.cy.js:29]: This test won't be retryable. Can you please create account dynamically? You may sample account creation tests from `cypress/e2e/wip/Tradershub`.

**Review by bilalmajid-deriv — COMMENTED** [review]
The Changes are looking good apart from retryablility issue as highlighted by @abuzar-deriv

File: cypress/e2e/wip/Crypto/crypto_account_check.cy.js (hunk 1 above)
Comment (vimalan-deriv) [reply cypress/e2e/wip/Crypto/crypto_account_check.cy.js]: Have added the retry mechanism please help to check on it again, Thank you

File: cypress/e2e/wip/Crypto/crypto_account_check.cy.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
+import '@testing-library/cypress/add-commands'
+import cryptoconfig from './pageobjects/common.js'
+import { generateEpoch } from '../../../support/tradersHub'
```
Comment (abuzar-deriv) [inline cypress/e2e/wip/Crypto/crypto_account_check.cy.js:3]:
```suggestion
import { generateEpoch } from '../../../support/helper/utility'
```

File: cypress/e2e/wip/Crypto/crypto_account_check.cy.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
```
Comment (abuzar-deriv) [inline cypress/e2e/wip/Crypto/crypto_account_check.cy.js:3]:
```suggestion
import { generateEpoch } from '../../../support/helper/utility'
```

**Review by abuzar-deriv — APPROVED** [review]
Looks good.
PS: @vimalan-deriv , package-lock.json has just the whitespace change so I'm ignoring it but please try not to commit it unless the changes are intentional.

**Review by bilalmajid-deriv — APPROVED** [review]
LGTM

**Review by markodonnell-deriv — APPROVED** [review]
Looks good

**Pull request merged**